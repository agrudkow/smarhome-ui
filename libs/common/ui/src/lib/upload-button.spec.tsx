import React from 'react';
import { render } from '@testing-library/react';

import UploadButton from './upload-button';

describe(' UploadButton', () => {
  it('should render successfully', () => {
    const { baseElement } = render(<UploadButton />);
    expect(baseElement).toBeTruthy();
  });
});
