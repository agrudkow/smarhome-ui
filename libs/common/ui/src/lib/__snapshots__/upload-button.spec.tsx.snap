// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` UploadButton should match snapshot 1`] = `
.c0 {
  width: 100%;
  font-weight: 550;
  white-space: nowrap;
  background-color: #4E4E4E;
  color: #E3E2E1;
  opacity: 0.7;
  -webkit-transition: color 0.3s ease-in-out,background-color 0.3s ease-in-out;
  transition: color 0.3s ease-in-out,background-color 0.3s ease-in-out;
}

.c0:hover {
  background-color: #1BBDCE;
  color: #102128;
}

<div>
  <input
    accept=".txt"
    id="contained-button-file"
    style="display: none;"
    type="file"
  />
  <label
    for="contained-button-file"
  >
    <span
      aria-disabled="false"
      class="MuiButtonBase-root MuiButton-root MuiButton-contained c0 MuiButton-containedPrimary"
      role="button"
      tabindex="0"
    >
      <span
        class="MuiButton-label"
      >
        UPLOAD
      </span>
      <span
        class="MuiTouchRipple-root"
      />
    </span>
  </label>
</div>
`;
